users_notification_settings.sql

```diff
@@ -0,0 +1,42 @@
+-- This is just a test (in draft) file so that Arit and Ridhwana can work together using revision control
```

Comment:
is this going to be moved in a data update script as well?

Reply:
I will check and update 👍🏾